index.html

```diff
@@ -0,0 +1,216 @@
+<html>
+
+<head>
+    <title>Bike Sharing Project - IEEE NITK</title>
+    <!-- Compiled and minified CSS -->
+    <!-- Load Core Materialize CSS  -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/css/materialize.min.css">
+    <link rel="stylesheet" href="/static/css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="shortcut icon" href="/static/img/logo.png" type="image/x-icon">
+    <link rel="stylesheet" href="style1.css">
```

Comment:
Please check if the file path is correct when you are running the Flask app. There is already a styles.css file present in the CSS Directory! I don't see the point of creating another CSS File there!